python/pfs/drp/stella/createDetGeom.py

```diff
@@ -0,0 +1,108 @@
+import os
+import sys
+import argparse
+
+import lsst.pex.config                  as pexConfig
+import lsst.afw.table as afwTable
+import lsst.afw.geom as afwGeom
+from lsst.pipe.base import Task
+
+class CreateDetGeomConfig(pexConfig.Config):
+    """Configuration for creating the PFS detector geometry files"""
+
+class CreateDetGeomTask(Task):
+    """Task to create the detector geometry files for PFS"""
+    ConfigClass = CreateDetGeomConfig
+    _DefaultName = "createDetGeomTask"
+
+    def __init__(self, *args, **kwargs):
+        super(CreateDetGeomTask, self).__init__(*args, **kwargs)
+
+    def run(self):
+        gain = (1.24, 1.24, 1.27, 1.18, 1.26, 1.20, 1.24, 1.26)
+        rdnoise = (3.61, 3.78, 3.18, 2.95, 3.19, 3.80, 4.51, 3.18)
+        
+        outDir = os.getenv('OBS_PFS_DIR', '~/stella_git/obs_pfs')
+        print 'outDir = <',outDir,'>'
+        outDir = outDir+'/pfs/camera'
```

Comment:
At one point, this was configurable (albeit with an arguably unfortunate default). Now it's that. Why the change?

Actually, that makes me question something more fundamental: while I've not yet fully understood the structure of the PFS codebase, this script seems like something that relates directly to the PFS camera. Why doesn't it live in obs_pfs rather than drp_stella?

Finally, `os.path.join()` rather than string concatenation, and `lsst.utils.base.getPackageDir()` rather than `os.getenv()`.